recipes/algotom/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "algotom" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/algotom/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
Any reason to not use the PyPI source?

Reply:
Just because I modified the recipe from the one I used before. I can change to pypi